CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -50,7 +50,7 @@ Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	1 Action
 Range:		40m
-Effect:		16 % chance to miss. Deal 52 damage.
+Effect:		2 chance to miss. Deal 52 damage. AP level 3
 If target has no shield left and is still alive, apply a 
 poison payload of your choice.
```

Comment:
Fast Spike is purely better than Harmacist Spike, is that intentional?

Also, why does the text say 'no shield left' vs 'if pierces armor'? 
There's a couple of questions this brings... Normally conventional weapons are not stopped by most shields, so why does this reference shields when it isn't blocked by shields?
And is your intention that the needles will always apply poison regardless of piercing armor or not (because that's how it reads to me)?

Reply:
It is an intentional improvement. I should probably have Harmacist spike improve at level 5 to offer a difference in kind.

It is my intention that if the needle does not defeat armor, that no poison is delivered. I will adjust the language.